WasatchNET/Spectrometer.cs

```diff
@@ -1990,7 +2007,15 @@ protected virtual double[] getSpectrumRaw()
                         try
                         {
                             // read all expected pixels from the endpoint
-                            subspectrum = readSubspectrum(spectralReader, pixelsPerEndpoint);
+                            if (isStroker)
+		                    {
+                    		    subspectrum = readSubspectrumStroker(spectralReader, pixelsPerEndpoint);
+                    		    if (areaScanEnabled)
+                        	        pixelsPerEndpoint *= 70;

```

Comment:
You're going to have to explain this one in a comment.  Is this hard-coding the number of detector rows?  And wouldn't it be 64 for most 2D Hamamatsu detectors?  And given the number of places you reference 70, please make it a constant.